src/converters/monitor.ts

```diff
@@ -145,6 +145,10 @@ export class NumericCharacteristicMonitor extends BaseCharacteristicMonitor {
     }
     const percentage = (input - this.input_min) / (this.input_max - this.input_min);
 
-    return out_minimum + percentage * (out_maximum - out_minimum);
+    const result = out_minimum + percentage * (out_maximum - out_minimum);
+    if (result < 1 && result > 0) {
+      return Math.ceil(result);
```

Comment:
Shouldn't we simply just return 1 here, as we already know the outcome of `Math.ceil` in this case because of the `if` condition.
```suggestion
      return 1;
```

Reply:
Oh sorry, I didn't check properly. This shouldn't be a fixed value. I think we definitely need a test to figure this one out.

I would expect the condition to some how relate to the `out_minimum` boundary (and perhaps we need the same for the `out_maximum`).

Reply:
You‘re right. That statement will always return 1 within the _if_ condition. Why not remove the condition and apply _Math.ceil_ always to the result? Is that function ever going to output a decimal number?